app/src/main/java/com/kickstarter/libs/utils/extensions/AnyExt.kt

```diff
@@ -0,0 +1,54 @@
+package com.kickstarter.libs.utils.extensions
+
+import com.kickstarter.libs.utils.ObjectUtils
+import io.reactivex.functions.Function
+import rx.functions.Func1
+
+fun <T : Any> T?.isNull(): Boolean {
+    return this == null
+}
+
+fun <T : Any> T?.isNotNull(): Boolean {
+    return this != null
+}
+
+fun <T : Any> T?.coalesce(theDefault: T): T {
+    if (this.isNotNull()) {
+        return this!!
+    }
+    return theDefault
+}
+
+fun <T : Any> coalesceWith(theDefault: T): Func1<T, T> {
+    return Func1 { value: T -> ObjectUtils.coalesce(value, theDefault) }
```

Comment:
Here substitute the `ObjectUtils.coalesce(value, theDefault)` over the `value.coalesce(theDefault)` to use the extension method on line 15

Reply:
Woops, didn't realize i left that there 😅😅